service/k8sclient/k8client.go

```diff
@@ -502,6 +528,14 @@ func (c *K8Client) getPerconaXtraDBClusters(ctx context.Context) ([]XtraDBCluste
 				ComputeResources: c.getComputeResources(cluster.Spec.ProxySQL.Resources),
 			},
 			Pause: cluster.Spec.Pause,
+			DetailedState: &DetailedState{
+				status: []appStatus{
+					appStatus{cluster.Status.PMM.Ready, cluster.Status.PMM.Size},
```

Comment:
Make ensure that we have these fields and we won't get panic

Reply:
Well, those fields are not pointers so the worst that can happen is we get zero value for Ready and Size, right?